CreeDictionary/CreeDictionary/templates/CreeDictionary/components/metatags.html

```diff
@@ -0,0 +1,35 @@
+{% spaceless %}
+  {% comment %}
+    Inserted into the <head>, this adds tags for social media preview.
+
+    Parameters:
+     - title — the title of the webpage -- as this goes in the tab bar, it's
+       the link in search results, and is visible in social media links is the
+       MOST important part!
+     - description — a short summary of the current page
+  {% endcomment %}
+
+  {% load url_extras %}
+
+  {% with title_="itwêwina: the online Cree dictionary" desc_="A smart, online dictionary of Plains Cree (Y-Dialect) | nêhiyawi-masinahikan kâ-iyinîsit" %}
```

Comment:
@aarppe please suggest here what the defaults should be!

Reply:
```suggestion
  {% with title_="itwêwina: the online Cree dictionary" desc_="A smart, online dictionary of Plains Cree (Y-Dialect) | nêhiyawi-masinahikan kâ-iyiniwâk" %}
```

Or I think Antti also suggested "nêhiyaw-itwêwina ayapîhk"